deployments/deployment.yaml

```diff
@@ -224,5 +228,11 @@ parameters:
   - name: MEMORY_REQUESTS
     value: 100Mi
   - name: LOGGING_LEVEL
-    value: warn
+    # value: warn
+    value: debug
```

Comment:
should this be changed back to warn before merging? just wanted to point this out so it's not forgotten about.

Reply:
Yes it should be changed! many thanks!